src/wp-includes/blocks/calendar.php

```diff
@@ -40,11 +40,32 @@ function render_block_core_calendar( $attributes ) {
 		}
 	}
 
+	$color_block_styles = array();
+
+	// Text color.
+	$preset_text_color          = array_key_exists( 'textColor', $attributes ) ? "var:preset|color|{$attributes['textColor']}" : null;
+	$custom_text_color          = _wp_array_get( $attributes, array( 'style', 'color', 'text' ), null );
+	$color_block_styles['text'] = $preset_text_color ? $preset_text_color : $custom_text_color;
+
+	// Background Color.
+	$preset_background_color          = array_key_exists( 'backgroundColor', $attributes ) ? "var:preset|color|{$attributes['backgroundColor']}" : null;
+	$custom_background_color          = _wp_array_get( $attributes, array( 'style', 'color', 'background' ), null );
+	$color_block_styles['background'] = $preset_background_color ? $preset_background_color : $custom_background_color;
+
+	// Generate color styles and classes.
+	$styles        = gutenberg_style_engine_get_styles( array( 'color' => $color_block_styles ), array( 'convert_vars_to_classnames' => true ) );
```

Comment:
@ockham The prefix should be changed to `wp_` instead of `gutenberg_`.

Reply:
Ugh, I missed that one. I'll fix it.

Reply:
Sidenote, I don't quite understand how this ended up here. The relevant GB PR (https://github.com/WordPress/gutenberg/pull/42029) was only [merged](https://github.com/WordPress/gutenberg/commit/dc57581be214d5c2e6b77cfabf06dc75ffc374cf) after GB 14.1 was released, and we didn't cherry-pick it to the `wp/6.1` branch. In fact, the relevant code is absent from [that file in Gutenberg's `wp/6.1` branch](https://github.com/WordPress/gutenberg/blob/wp/6.1/packages/block-library/src/calendar/index.php) 🤔

Reply:
Looks like the script picked up the wrong package version(s): [For `@wordpress/block-library`, I see `7.15.0`](https://github.com/WordPress/wordpress-develop/pull/3342/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R86).

But the [`wp-6.1` tag points to `7.14.3`](https://www.npmjs.com/package/@wordpress/block-library).

I think we had a similar issue before with https://github.com/WordPress/wordpress-develop/pull/3154, but I'm not 100% sure what @gziolo did to fix it...

Reply:
(Re-syncing now.)